### PR TITLE
cleanup

### PR DESCRIPTION
**Pull request by mforets**

### CODE REVIEW

**Comment by mforets** [comment]
this should be ready to merge after the tests pass..

File: models/SinglePendulum/SinglePendulum.jl [hunk 1]
```diff
@@ -20,15 +17,17 @@
...
+# The state vector is ``[θ, ̇θ̇]``.
 # Controllers are trained using behavior cloning. Here, a neural network is
-# trained to replicate expert demonstrations.
+# trained to replicate expert demonstrations [^1].
 
 using NeuralNetworkAnalysis
 
-controller = read_nnet("./controller_single_pendulum.nnet");
+models_dir = normpath(@__DIR__, "..", "..", "..", "models")
+path = joinpath(models_dir, "SinglePendulum", "controller_single_pendulum.nnet")
```
Comment (schillic) [inline models/SinglePendulum/SinglePendulum.jl:27]: What does this do? I think we should have a macro like `@relpath` in `ReachabilityModels`.

Comment (mforets) [reply models/SinglePendulum/SinglePendulum.jl:27]: yes, that would be convenient..

**Comment by mforets** [comment]
the new commits addressed  your comments, also fixing the conflict warnings. i also fixed the naming convention for the models. they are now sorted as they appear in the arch-comp article.

File: docs/generate.jl [hunk 2]
```diff
@@ -2,20 +2,21 @@
...
-    joinpath(@__DIR__, "..", "models", "Double Pendulum"),
-    joinpath(@__DIR__, "..", "models", "VertCAS")
+    joinpath(@__DIR__, "..", "models", "Cart-Pole")
 ]
 GENERATEDDIR = joinpath(@__DIR__, "src", "models")
 mkpath(GENERATEDDIR)
+
 for model in MODELS
-    # copy over .nnet files
+    # copy over .nnet files << TODO still needed?
```
Comment (schillic) [inline docs/generate.jl:19]: Do you want to fix that here?

**Review by schillic — APPROVED** [review]
Sometimes the "solve it" is in the "Model" section and sometimes in the "Results" section.

File: docs/generate.jl (hunk 2 above)
Comment (mforets) [reply docs/generate.jl:19]: i wasn't sure if we can remove it. i suppose that the answer is yes (store the `.nnet` files in the `models` folder and use `@relpath`

**Comment by schillic** [comment]
There are problems in the docs for the Airplane model at the bottom.

**Pull request merged**